### PR TITLE
environment_check.sh: fix quoting and add timeout

### PR DESCRIPTION
**Pull request by kaxing**
- add missing double quotes to a error message
- avoid waiting really long time on sles/zypper for checking packages
- remove unneeded single quotes

context: https://github.com/longhorn/longhorn/issues/3550#issuecomment-1117525837

### CODE REVIEW

**Review by innobead — APPROVED** [review]
LGTM

**Comment by innobead** [comment]
cc @derekbit

File: deploy/scripts/environment_check.sh [hunk 1]
```diff
@@ -90,7 +90,7 @@ set_packages_and_check_cmd()
     PACKAGES=(nfs-utils iscsi-initiator-utils)
     ;;
   *"suse"* )
-    CHECK_CMD='zypper se -i'
+    CHECK_CMD='rpm -q'
```
Comment (derekbit) [inline deploy/scripts/environment_check.sh:93]: Just learned SLES also supports `rpm` in your commit. :)

**Review by derekbit — COMMENTED** [review]
LGTM, thanks.

**Pull request merged**